### PR TITLE
fishing lvls?

### PR DESCRIPTION
**Pull request by Iamgoofball**
selling addy 2h 10gp

Adds level up sounds for the 4 skills you can level, along with 2 extra for when I get around to those.

Cooking is now a used skill.

Fishing is now a used skill.

Overhauls fishing.

There's now 6 types of fishing tools available, and the more levels you get the better fish you can catch.

You now get experience from cooking crap, and the better fish you cook the better experience you get.

### CODE REVIEW

File: code/modules/agriculture/fishing.dm [hunk 1]
```diff
@@ -3,108 +3,123 @@
...
-	name = "Lobster"
-	desc = "Pinchy!"
-	icon_state = "lobster"
-
-/obj/item/food/fish/crab
-	name = "Crab"
-	desc = "Pinchy Jr."
-	icon_state = "crab"
-
-/obj/item/food/fish/sponge
```
Comment (RemieRichards) [inline code/modules/agriculture/fishing.dm]: NO

Comment (Iamgoofball) [inline code/modules/agriculture/fishing.dm]: Give me the following for it and I can readd it:

Required Fishing Level
Fishing Experience Granted
Required Cooking Level
Cooking Experience Granted

**Comment by sawu-tg** [comment]
@Iamgoofball  Do not remove a large amount of features in way of adding lesser usable features. Add back the old dynamic fish into the new system, and add back fishable areas. Also, buoys were effects for a reason. If you don't add more redundancy clauses to prevent random floating buoys, thats also a no-go.

Make a new PR when everything is up to standard.

**Comment by Iamgoofball** [comment]
@sawu-tg Mind just reopening this so I don't have to make a new branch and shit?

Also, I kept trying to ping you in the irc, you didn't respond.

I'll work the other fishes into the available tools

**Comment by DaedalusGame** [comment]
i must get my PR merged before goof so goof has a bunch of merge conflicts to resolve

File: code/world/world.dm [hunk 2]
```diff
@@ -37,8 +37,8 @@ var/datum/controller/balance/balance
 				playerValidHair |= i
 		messageSystemAll("GENERATING WORLD..")
 		spawn(1)
-			for(var/i = 1; i < world.maxz; i++)
-				generate(i)
+		//for(var/i = 1; i < world.maxz; i++)
```
Comment (sawu-tg) [inline code/world/world.dm]: Remember to uncomment code you comment for testing.

File: code/object/object_types/resources/skill/woodcutting.dm [hunk 3]
```diff
@@ -26,6 +26,7 @@
 		if(required_level > user.playerData.firemaking.statCurr)
 			messagePlayer("You aren't good enough at firemaking to light a fire here.",user,src)
 			return
+		new world
```
Comment (sawu-tg) [inline code/object/object_types/resources/skill/woodcutting.dm]: What is this and why is it even here.

File: code/world/world.dm (hunk 2 above)
Comment (Iamgoofball) [inline code/world/world.dm]: i shouldn't of committed this file, whoops

File: code/object/object_types/resources/skill/woodcutting.dm (hunk 3 above)
Comment (Iamgoofball) [inline code/object/object_types/resources/skill/woodcutting.dm]: i shouldn't of committed this file, whoops

**Pull request merged**